packages/create-bison-app/template/.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+/.data
```

Comment:
Are there any more folders that should be added here? Basically you'd want to exclude anything that isn't needed to run the app (e.g. tests). Ideally all of the app code would be in one folder (e.g. `src`) and we could just mount that folder in docker and not need a `.dockerignore` file, but Bison's organized a bit differently.